### PR TITLE
Feat/lifecycle app

### PR DESCRIPTION
**Pull request by sidohaakma**

### CODE REVIEW

File: apps/catalogue/package.json [hunk 1]
```diff
@@ -8,11 +8,23 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^1.2.35",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
+    "@fortawesome/vue-fontawesome": "^2.0.2",
+    "@molgenis-ui/components-library": "^2.4.5",
     "@mswertz/emx2-styleguide": "*",
     "core-js": "3.11.2",
+    "bootstrap-vue": "^2.21.2",
```
Comment (connoratrug) [inline apps/catalogue/package.json:17]: i thought @mswertz idea was to use a custom build comp lib ? , i'm not against using bs-vue but is does lock us into the bs-vue - release cycle.

File: apps/catalogue/src/views/lifecycle/MappingView.vue [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+    ...mapActions(['fetchCohorts', 'fetchMappings']),
+    getMatchStatus(variableName, databankAcronym) {
+      if(!this.mappings[variableName] || !this.mappings[variableName][databankAcronym]) {
+        return 'danger' // not mapped
+      }
+
+      const match = this.mappings[variableName][databankAcronym]
+      switch(match) {
+        case 'zna':
+        case 'zna':
```
Comment (connoratrug) [inline apps/catalogue/src/views/lifecycle/MappingView.vue:46]: duplicate line

File: apps/catalogue/package.json [hunk 3]
```diff
@@ -8,11 +8,23 @@
...
+    "highlight.js": "^10.7.2",
     "vue": "2.6.12",
-    "vue-router": "3.5.1"
+    "vue-multilevel-accordion": "^1.0.8",
+    "vue-router": "3.5.1",
+    "vuedraggable": "^2.24.3",
+    "vuex": "^3.6.2"
+  },
+  "devDependencies": {
+    "tailwindcss": "^1.1.4"
```
[the remark this answers is not in the document]
Comment (sidohaakma) [reply apps/catalogue/package.json:27]: tailwind was needed for the tree you suggested, but this breaks the emx2 css so I removed it again but forgot to drop this one.

File: apps/catalogue/package.json (hunk 1 above)
Comment (sidohaakma) [reply apps/catalogue/package.json:17]: I think it is a good idea to lean on the filters we already made and pull them into this repository. It is a nice framework which can help us a lot. The filters we created for dataexplorer 2 require bootstrap vue

File: apps/catalogue/package.json (hunk 3 above)
[the remark this answers is not in the document]
Comment (connoratrug) [reply apps/catalogue/package.json:27]: i did not suggest using the lib, i shared the screenshot to show how they addressed the tree ux issues , the tree component added to this PR does not really address of deep trees and horizontal space.

**Pull request closed without merging**